src/schema/v2/partner/Settings/updatePartnerContactMutation.ts

```diff
@@ -0,0 +1,142 @@
+import {
+  GraphQLBoolean,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLUnionType,
+} from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import {
+  formatGravityError,
+  GravityMutationErrorType,
+} from "lib/gravityErrorHandler"
+import { ContactType } from "schema/v2/Contacts"
+import { ResolverContext } from "types/graphql"
+
+interface UpdatePartnerContactInputProps {
+  contactId: string
+  partnerId: string
+  name?: string
+  position?: string
+  canContact?: boolean
+  email?: string
+  phone?: string
+  locationId?: string
+}
+
+const SuccessType = new GraphQLObjectType<any, ResolverContext>({
+  name: "UpdatePartnerContactSuccess",
+  isTypeOf: (data) => !!data._id,
+  fields: () => ({
+    partnerContact: {
+      type: ContactType,
+      resolve: (result) => result,
+    },
+  }),
+})
+
+const FailureType = new GraphQLObjectType<any, ResolverContext>({
+  name: "UpdatePartnerContactFailure",
+  isTypeOf: (data) => data._type === "GravityMutationError",
+  fields: () => ({
+    mutationError: {
+      type: GravityMutationErrorType,
+      resolve: (err) => err,
+    },
+  }),
+})
+
+const ResponseOrErrorType = new GraphQLUnionType({
+  name: "UpdatePartnerContactOrError",
+  types: [SuccessType, FailureType],
+})
+
+export const UpdatePartnerContactMutation = mutationWithClientMutationId<
+  UpdatePartnerContactInputProps,
+  any,
+  ResolverContext
+>({
+  name: "UpdatePartnerContact",
+  description: "Updates an existing contact for a partner",
+  inputFields: {
+    contactId: {
```

Comment:
What are your thoughts on input id naming convention?

`contactId` vs `contactID`? (I'll make this consistent with my previous PRs in a clean up pass)

Reply:
We're all over the place in MP - I wouldn't worry too much about it. Because of the types/Relay support - you'll get the appropriate error/auto-complete if you try to use one when a mutation expects the other.

We should be consistent within a single mutation (just for a bit of sanity - why not), but in general in MP we have lots of examples of both formats.